### ISSUE TITLE
AVT 1 - Number Input has DAP violations 

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4532]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React
DAP IBM Accessibility 1.8.0.0 Sept 2019 Ruleset

## Detailed Description
1. Go to the React Component and select the disable knob
2. Run DAP and the following violations are present (see screenshot):
<img width="928" alt="Screen Shot 2019-11-01 at 3 13 08 PM" src="https://user-images.githubusercontent.com/21676914/68053815-ddfd6a80-fcba-11e9-83d5-13d2bc12c404.png">
3. Select the form validation knob and run DAP again and another violation is present:
<img width="990" alt="Screen Shot 2019-11-01 at 3 16 15 PM" src="https://user-images.githubusercontent.com/21676914/68053863-fd949300-fcba-11e9-968f-da02f6d16b8d.png">
